extensions/cornerstone/src/panels/PanelMeasurement.tsx

```diff
@@ -32,69 +42,50 @@ export default function PanelMeasurementTable({
       return;
     }
 
-    const measurements = [...displayMeasurements];
-    const measurement = measurements.find(m => m.uid === uid);
-
-    measurements.forEach(m => (m.isActive = m.uid !== uid ? false : true));
-    measurement.isActive = true;
-  };
-
-  const jumpToImage = (uid: string) => {
-    measurementService.jumpToMeasurement(viewportGrid.activeViewportId, uid);
-    onMeasurementItemClickHandler(uid, true);
-  };
-
-  const removeMeasurement = (uid: string) => {
-    measurementService.remove(uid);
+    displayMeasurements.forEach(m => (m.isActive = m.uid === uid));
   };
 
-  const renameMeasurement = (uid: string) => {
-    jumpToImage(uid);
-    const labelConfig = customizationService.get('measurementLabels');
-    const measurement = measurementService.getMeasurement(uid);
-    showLabelAnnotationPopup(measurement, uiDialogService, labelConfig).then(val => {
-      measurementService.update(
-        uid,
-        {
-          ...val,
-        },
-        true
-      );
-    });
-  };
-
-  const changeColorMeasurement = (uid: string) => {
-    const { color } = measurementService.getMeasurement(uid);
-    const rgbaColor = {
-      r: color[0],
-      g: color[1],
-      b: color[2],
-      a: color[3] / 255.0,
-    };
-    colorPickerDialog(uiDialogService, rgbaColor, (newRgbaColor, actionId) => {
-      if (actionId === 'cancel') {
-        return;
+  const bindCommand = (name: string, callJump = false) => {
+    return (uid: string) => {
+      if (!uid) {
+        debugger;
       }
-
-      const color = [newRgbaColor.r, newRgbaColor.g, newRgbaColor.b, newRgbaColor.a * 255.0];
-      // segmentationService.setSegmentColor(viewportId, segmentationId, segmentIndex, color);
-    });
+      if (callJump) {
+        commandsManager.runCommand('jumpToMeasurement');
+      }
+      commandsManager.runCommand(name, { uid });
+      if (callJump === null) {
+        onMeasurementItemClickHandler(uid, true);
+      }
+    };
   };
 
```

Comment:
We could implement a pipe operator in the `commandsManager` to allow chaining like this:

`commandsManager.run('toggleLockMeasurement|activateMeasurement|whatever')`

Otherwise, we would need to implement `bindCommand` everywhere, which involves custom logic for handling `callJump` and other parameters.

What do you think?

Reply:
When we need it, I'm going to implement a runCommands command which runs the specified commands.  That will allow chaining appropriately - in fact, I will do that now, since we need it.